platform-tests/src/test/java/org/junit/platform/engine/support/store/NamespacedHierarchicalStoreTests.java

```diff
@@ -331,41 +334,62 @@ void valueFromParentCanBeOverriddenInChild() {
 	@Nested
 	class CompositeNamespaceTests {
 
-		@Test
-		void namespacesEqualForSamePartsSequence() {
-			Namespace ns1 = Namespace.create("part1", "part2");
-			Namespace ns2 = Namespace.create("part1", "part2");
-
-			assertEquals(ns1, ns2);
-		}
-
-		@Test
-		void orderOfNamespacePartsDoesMatter() {
-			Namespace ns1 = Namespace.create("part1", "part2");
-			Namespace ns2 = Namespace.create("part2", "part1");
-
-			assertNotEquals(ns1, ns2);
-		}
-
 		@Test
 		void additionNamespacePartMakesADifference() {
 
-			Namespace ns1 = Namespace.create("part1", "part2");
-			Namespace ns2 = Namespace.create("part1");
-			Namespace ns3 = Namespace.create("part1", "part2");
+			String ns1 = "part1/part2";
+			String ns2 = "part1";
 
 			Object value2 = createObject("value2");
 
 			parentStore.put(ns1, key, value);
 			parentStore.put(ns2, key, value2);
 
 			assertEquals(value, store.get(ns1, key));
-			assertEquals(value, store.get(ns3, key));
 			assertEquals(value2, store.get(ns2, key));
 		}
 
 	}
 
+	@Nested
+	class CloseActionTests {
```

Comment:
Also missing a test for a element that threw an exception.

Reply:
Added in 4d00cc38d0f12f1c69386708e37309def8c1e839.